src/components/Card/Card.styles.ts

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import { EStatuses } from './Card.types';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const CardContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  padding: 5px 10px;
+  background-color: ${({ theme }) => theme.colors.primary};
+  gap: 5px;
+  border-radius: 10px;
+`;
+
+export const CardBadgeContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 3px;
+`;
+
+export const CardBadge = styled.div`
+  background-color: ${(props) => props.color || props.theme.colors.grey};
+  display: flex;
+  align-items: center;
+  flex-basis: 25%;
+  text-align: center;
+  justify-content: center;
+  border-radius: 5px;
+`;
+
+export const CardBadgeText = styled.p`
+  font-weight: 600;
+  font-size: 8px;
+  text-transform: uppercase;
+  color: black;
+`;
+
+export const CardTitle = styled.p`
+  font-weight: 400;
+  font-size: 18px;
+  color: ${({ theme }) => theme.colors.text};
+`;
+
+export const CardDescription = styled.p`
+  font-weight: 100;
+  font-size: 14px;
+  color: ${({ theme }) => theme.colors.text};
+`;
+
+export const CardFooter = styled.div`
+  display: flex;
+  // justify-content: space-around;
+  flex-direction: row;
+  align-items: center;
+`;
+
+export const DateTimeContainer = styled.div<{ status: EStatuses }>`
```

Comment:
I haven't used styled components but looks like that not-dom props should be named with `$` prefix, see

![image](https://github.com/CaptMelancholy/uni-pet-project/assets/1639962/ca115660-5cc2-4f88-a76a-98e32e8b2189)

and 

![image](https://github.com/CaptMelancholy/uni-pet-project/assets/1639962/6f58886b-4385-4c04-9469-0b39f81a463d)